index.html

```diff
@@ -286,20 +286,65 @@ <h3 class="reviews__title">
             <div class="feedback__inner">
                 <img src="img/Rectangle 38.png" alt="" class="feedback__images">
                 <form class="feedback__form">
-<h3 class="feedback__title">
-    Остались вопросы?
-</h3>
-<p class="feedback__text">
-    Мы перезвоним Вам и проконсультируем по любому вопросу
-</p>
-<input class="feedback__input" type="text" placeholder="Имя">
-<input class="feedback__input" type="text" placeholder="Телефон">
-<button class="feedback-btn" type="submit">Оставить заявку</button>
+                    <h3 class="feedback__title">
+                        Остались вопросы?
+                    </h3>
+                    <p class="feedback__text">
+                        Мы перезвоним Вам и проконсультируем по любому вопросу
+                    </p>
+                    <input class="feedback__input" type="text" placeholder="Имя">
+                    <input class="feedback__input" type="text" placeholder="Телефон">
+                    <button class="feedback-btn" type="submit">Оставить заявку</button>
                 </form>
             </div>
         </div>
     </section>
 
+    <footer class="footer">
+        <div class="container">
+            <div class="footer__top">
+                <a href="#" class="content-btn">Скачать приложение</a>
+                <nav class="menu">
+                    <ul class="menu__items">

```

Comment:
You can't include tag <li> into <a>. <li> should be inside <ul>. Correct structure => `<ul class="menu__items"><li class="menu__item"><a class="menu__item-link">`